.circleci/config.yml

```diff
@@ -0,0 +1,44 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:11.10.1
+
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      # - image: circleci/mongo:3.4.4
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "package.json" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
+      - run: npm install
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-{{ checksum "package.json" }}
+      # run tests!
+      - run:
+          name: run test
+          command: npm test
+      - run:
+          name: run tests with JUnit as reporter
+          command: npm run jest:junit
```

Comment:
I'm confused? Who's using JUnit and how is this part of your task?

Reply:
jUnit is how circle ci gets its own report of the test.

The first image has junit configured and the second image doesn't

<img width="1591" alt="Screenshot 2019-08-27 at 09 15 14" src="https://user-images.githubusercontent.com/39800005/63754038-16905a80-c8ac-11e9-98e5-99a2aad3d88b.png">
<img width="1595" alt="Screenshot 2019-08-27 at 09 15 41" src="https://user-images.githubusercontent.com/39800005/63754072-227c1c80-c8ac-11e9-900e-3807a7df5340.png">

Reply:
Really nice. So instead of just running it in the logs and sending it to code climate, we'll also see the report here?

Reply:
Yes